app/views/extended_metadata_types/_emt_tabs.erb

```diff
@@ -0,0 +1,156 @@
+<% nested, top_level = @extended_metadata_types.partition(&:extended_type?).each{ |group| group.sort_by!(&:id).reverse! } %>
+
+<% if ExtendedMetadataType.disabled_but_in_use.any? %>
+  <div class="alert alert-warning">
+    <p>
+      Extended metadata types that are in use have been disabled, and are highlighted below.
+    </p>
+    <p>
+      The existing metadata won't be lost but will be hidden and the metadata type unavailable for new items.
+    </p>
+  </div>
+<% end %>
+
+<ul class="nav nav-tabs" role="tablist">
+  <li class="nav-item active in">
+    <a class="nav-link" id="top-level-tab" data-toggle="tab" href="#top-level-metadata-table" role="tab" aria-controls="top-level" aria-selected="true" data-tooltip="This tab displays all top-level extended metadata types.">
+      <span class="help_icon"></span>Top Level
+    </a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link" id="nested-tab" data-toggle="tab" href="#nested-metadata-table" role="tab" aria-controls="nested" aria-selected="false" data-tooltip="This tab lists all extended metadata types with the supported type ExtendedMetadata.">
+      <span class="help_icon"></span>Nested Level
+    </a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link" id="sample-controlled-vocabs-tab" data-toggle="tab" href="#sample-controlled-vocabs" role="tab" aria-controls="sample-controlled-vocabs" aria-selected="false" data-tooltip="This tab lists all the controlled vocabularies (CV) available in SEEK">
+      <span class="help_icon"></span>Controlled Vocabs
+    </a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link" id="sample-attribute-types-tab" data-toggle="tab" href="#sample-attribute-types" role="tab" aria-controls="sample-attribute-types" aria-selected="false" data-tooltip="This tab provides a list of all supported extended metadata attribute types.">
+      <span class="help_icon"></span>Extended Metadata Attribute Types
+    </a>
+  </li>
+</ul>
+
+<div class="tab-content">
+  <div class="tab-pane fade active in" id="top-level-metadata-table" role="tabpanel" aria-labelledby="top-level-tab">
+    <div class="alert alert-info">
+      <ul>
+        <li>
+          You can add extended metadata to the following SEEK resource types:  <strong>Investigation</strong>, <strong>Study</strong>, <strong>Assay</strong>, <strong>Collection</strong>, <strong>DataFile</strong>, <strong>Document</strong>, <strong>Event</strong>, <strong>Model</strong>, <strong>Presentation</strong>, <strong>Sop</strong>, and <strong>Project</strong>.
```

Comment:
this should use `Seek::Config.instance_name` and the locale's for the type names. It could iterate over all the types with `supports_extended_metadata?` rather than hard coded